### ISSUE TITLE
twMediaDownloader not obtaining any tweets in Bookmarks added since March 31st, 2023 [Firefox / Chromium]

### ISSUE THREAD

**Issue by marceloexc** [furyutei/twMediaDownloader#110]
I wonder if there was an API change recently to the front end of the website, because from my understanding the twMediaDownloader extension for both Firefox [Firefox] and Chromium browsers [Tested on Brave] just use the instance from your browser tab.

Any tweet that I add to my "bookmarks" since March 30th, 2023, is ignored and will not get downloaded.

Gif of event: 
![demo twmediadownloader](https://user-images.githubusercontent.com/35786323/230457623-652f459b-2126-48d4-a2ad-85705905f952.gif)

**Comment by marceloexc** [comment]
I dug through this a little bit more, i downloaded the extension as a tampermonkey script on firefox and I saw some oauth keys, so I was wrong on the basis that twmediadownloader only uses your browser session and scrapes from your twitter tab locally through scraping the web elements (which is what gallery-dl does IIRC), and instead it does use the twitter api to access at least some functions, which Im assuming is connected to downloading bookmarked media.

Unfortunately, while I was trying to get some of my own oauth keys and edit it into the script, i saw that the starting price is $100 a month...outrageous...I do not blame the dev for not wanting to continue the project

Surely there must be a free tier, right?

**Comment by fireattack** [comment]
Unfortunately I don't think Elon Musk is going to give you anything for free in the future.

Also keep in mind, some of the auth tokens used in twmediadownloader are actually not normal tokens you can acquire from dev account. They're used by Twitter's official web application etc., and the API will return different stuff (usually better) if you use them.

**Comment by marceloexc** [comment]
Yeah i switched to gallery-dl for this purpose, instead of using oauth keys it uses my browser's cookies for authentication. I liked twMediaDownloaders filename structure so I formatted my twitter extractor on gallery-dl to be the same. However every week or so twitter changes something and it breaks

Funnily enough i tried to login to my twitter dev acc for a comp sci project i made around 3 years ago, with all scopes enabled + old twitter API, and was able to look at my twitter dev dashboard...however the moment I tried to do anything with it It got disabled. Lol
